modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/util/DepotAdminGroupSearchProvider.java

```diff
@@ -64,6 +64,25 @@ public GroupSearch getGroupSearch(
 		return _getGroupSearch(null, portletRequest, portletURL);
 	}
 
+	private List<Group> _digestGroups(Group group, List<Group> groups) {
```

Comment:
Can you rename the method to `_processGroups`? "digest" is normally used in the context of hashing, so it sounds weird here.

Reply:
Sure!

I'll wait for the content-management test to finish and then I'll rename that!